### ISSUE TITLE
"Error" cell contains an empty span element in file integrity table

### ISSUE THREAD

**Issue by audrasjb** [WordPress/health-check#267]
### Feature request/bug description

In file integrity table, the first cell of each error line is empty (it only contains an empty span used to generate a CSS content). There should at least be a screen-reader-text.

Current markup:

```
<tr>
   <td><span class="error"></span></td>
   <td>wp-admin/js/widgets.js</td>
   <td>Content changed <a href="#health-check-diff" data-file="wp-admin/js/widgets.js">(View Diff)</a></td>
</tr>
```

**Comment by Clorith** [comment]
Very good point, there's a few cases where an error or warning symbol is used, we should ensure all of them have a screen reader equivalent.

**Comment by Clorith** [comment]
Fixed in #278
